react/utils/client/generateHash.ts

```diff
@@ -0,0 +1,19 @@
+import {ArgumentNode, DocumentNode, visit} from 'graphql'
+
+const hashFromQuery = (asset: any) => ({
+  Argument(node: ArgumentNode) {
+    if (node.name.value === 'hash') {
+      asset.hash = node.value.value
+    }
+  }
+})
+
+export const generateHash = (query: DocumentNode) => {
+  if (query.documentId) {
+    return query.documentId
```

Comment:
What is this `documentId`? Is it the same as `hash`?

Reply:
Yes, it should be there for legacy compatibility